### ISSUE TITLE
Enhancement - Fire barrels should have a menu to burn stuff

### ISSUE THREAD

**Issue by pinkamedic** [loiste-interactive/Obenseuer-Issues#2819]
the fire barrel is a nice decorative piece but it would be better if you could throw items into it to turn the fire on / off, it should have a large space inside of it to collect all of the ashes from said burned items. it also needs to be craft able as right no it does not have a recipe and the only one that you can get is the one inside of the player tenement

**Comment by Pangaron** [comment]
Agree completely. I also think there could be a makeshift variant to make charcoal early (basically you need a barrel, one way vent for gasses and fire under it) - it would take a long time, like 12+ hour, but "baking" charcoal is really easy with makeshift tools (you just need heat, no oxygen, wood and way to releases gasses + patience).

**Comment by vulpes133** [comment]
I agree with giving fire barrels a use like this. Give us a way of disposing of all of our trash items we might accumulate in our house without needing to head through a loading screen to get outside and dump into the trash cans around town.
